pkgs/by-name/zi/zitadel/console.nix

```diff
@@ -42,7 +41,7 @@ stdenv.mkDerivation {
 
```

Comment:
The hash in line 31 must be updated, too. I got `sha256-UzmwUUYg0my3noAQNtlUEBQ+K6GVnBSkWj4CzoaoLKw=`